blog/content/post/2023-10-22-liquid-cooling.md

```diff
@@ -0,0 +1,147 @@
+I had to watch a few tutorials how to NOT mount the liquid cooling to make it
+perform at its best. Personally I recommend the following in general:
+
+- [The Gamer's Nexus' guide](https://www.youtube.com/watch?v=BbGomv195sk)
+- [The Provoked Prawn's guide](https://www.youtube.com/watch?v=qQcyYHGtArs)
+
+They give very good overview how to properly mount a liquid cooling. That's
+what I watched initially when getting to know how to deal with liquid cooling.
+
+### Mounting the liquid cooling
+
+Now that I have been promoted from total noob, to a beginner, I have unboxed
+the cooling parts and started mounting them. Having learned about the air
+bubbles and recommended mounting I have decided to mount it on the front side
+of the chassis with the pipes going upwards to the CPU cooling block. This
+should ensure that the pump will live a long life. Yes...
+
+That's where the hardships began. I own a Nvidia RTX 3060, which is quite a
+big monster to be honest. To my surprise, when I mounted every parts of the
+cooling according to my initial plan, it occurred that I can no longer fit my
+graphics card... The cooling pipes going from radiator to the CPu cooling
+block were too short to pass them around the Nvidia GPU. So either I had to
+put a smaller GPU or resign from liquid cooling. It was very bad news for me.
+
+But then I took another look at the radiator carefully and noticed, that the
+pump is actually not in the CPU cooling block, but in the radiator itself!
+
+![Pump and radiator](/img/liquid_cooling_radiator.jpg)
+
+Remembering physics back from the school, if I placed the radiator with the
+pipes on the upper side, the air bubbles would not get into the pump right? To
+be sure I have searched for a more suitable tutorial, specific to my liquid
+cooling model and I found [Gear Seeker's
+guide](https://www.youtube.com/watch?v=ayE9X71SDeY) which explicitly says that
+this cooling should be mounted with the pipes upside. You may imagine what
+relief I have felt when I watched it. I took apart the cooling from my chassis
+right away and started mounting it with the pipes on the upper side. THanks to
+that the pipes would no longer need to go around my GPU (they would actually
+be hanging above the card with plenty of free space).
+
+![Liquid cooling mounted](/img/liquid_cooling_mounted.jpg)
+
+### Cooling efficiency
+
+I have no comparison of liquid cooling on a i5-12600K, so I can only share the
+results of i9-14900K. When idling the CPU stays at around 40 Celsius degrees.
+
+![Idle CPU temperature](/img/cpu_idle_temp.png)
+
+When playing a not-so-heavy game like League of Legends for example, the
+temperature varies from 60 to 80 degrees depending on the load.
+
+![CPU temperature under semi load](/img/cpu_semi_load_temp.png)
+
+This is where the fans are getting a bit noisy. I am considering adjusting the
+fan curves to reduce the loudness. Also the pump is generating its own
```

Comment:
Have the pump speed follow CPU temp, and fan speed follow liquid temperature (if you have a temperature probe in there). This yields best results for me - quiet at idle, but cool under load :)

Reply:
Yeah gladly, but the pump does not have PWM pin (3pins only) so cannot be controlled. Maybe it could be controlled by voltage, but I am not sure if the pump will like undervoltage.